### DISCUSSION TITLE
Joomla Compatibility

### DISCUSSION THREAD

**Discussion by SirioAstarot** [elementor/elementor#18010] [Developers Community]
I have tested this component on WP and it is great, now I use Joomla because it is the best CMS on the market, I would like to use Elementor and its extensions with Joomla 4, adding this compatibility would attract many CMS users.

**Comment by WebsiteDons** [comment]
I honestly don't see by the statistical numbers how Joomla, in your opinion, is classified as "best CMS".

However, there are a few visual composition extensions available for Joomla and the currently preferred is SP Page Builder from JoomShaper.
![2022-05-04 13_35_24](https://user-images.githubusercontent.com/42153624/166746405-39ab5f1d-f65a-48eb-9133-4bd8b2f34211.jpg)
